src/main/java/org/folio/circulation/support/CqlOrder.java

```diff
@@ -0,0 +1,24 @@
+package org.folio.circulation.support;
+
+public class CqlOrder {
+
+  private final String index;
+  private final CqlSortDirection direction;
+
+  public CqlOrder(String index, CqlSortDirection direction) {
+    this.index = index;
+    this.direction = direction;
+  }
+
+  public static CqlOrder asc(String index) {
```

Comment:
Personally, I'd be tempted to use the full words here. Is there a reason to keep them short?

Reply:
Renamed to full words